iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.hpp

```diff
@@ -186,10 +178,10 @@ class storable_function<Capacity, signature<ReturnType, Args...>>
 
     // we need these templates to preserve the actual CallableType for the underlying call
     template <typename CallableType>
-    static void copy(const storable_function& src, storable_function& dest) noexcept;
+    static void copy_without_destroy(const storable_function& src, storable_function& dest) noexcept;
```

Comment:
Oh no, the assignment operator does not call this directly but by the `operations::copy` indirection. I think it would be better to have it consistent, so either change it back or also change these names. Up to you. Sorry to bother you